.github/workflows/test_component.yml

```
@@ -0,0 +1,87 @@
+name: Test component
+
+on:
+  workflow_call:
+    inputs:
+      artifact_run_id:
+        type: string
+        default: ""
+      amdgpu_families:
+        type: string
+      test_runs_on:
+        type: string
+      platform:
+        type: string
+      component:
+        type: string
+
+
+permissions:
+  contents: read
+
+jobs:
+  test_component:
+    name: 'Test ${{ fromJSON(inputs.component).job_name }} (shard ${{ matrix.shard }} - ${{ fromJSON(inputs.component).total_shards }})'
```

How about `shard 1/4` instead of `shard 1 - 4`?
```suggestion
    name: 'Test ${{ fromJSON(inputs.component).job_name }} (shard ${{ matrix.shard }}/${{ fromJSON(inputs.component).total_shards }})'
```

---

Ha that was actually my first thought too. Must be a GH action job name bug, but it cuts off anything before the `/`.

Check it out here: https://github.com/ROCm/TheRock/actions/runs/17840864292/job/50730524042

I am considering `1 of 4` but `-` is more concise

---

Ah, good point. Unless there's a way to escape the `/` (that doesn't somehow break other things), I'd vote for `1 of 4`.